### PR TITLE
feat/DAT-2741:updates to molecular_test node

### PR DESCRIPTION
**Pull request by tlicht3**
https://jira.opensciencedatacloud.org/browse/DAT-2741

### CODE REVIEW

File: gdcdictionary/schemas/molecular_test.yaml [hunk 1]
```diff
@@ -892,6 +915,17 @@ properties:
       - Unknown
       - Not Reported
       - Not Applicable
+      - MIB1
+      - MAML3 (HCMI)
```
Comment (wwysoc2) [inline gdcdictionary/schemas/molecular_test.yaml]: Is the "HCMI" in DAT-2741 just someone denoting who asked for this gene @tlicht3 ?

File: gdcdictionary/schemas/molecular_test.yaml [hunk 2]
```diff
@@ -980,6 +1028,33 @@ properties:
       - Total Protein
       - Unknown
       - Not Reported
+      - Epstein-Barr Virus
+      - Human Papillomavirus
+      - HPV-E6/E7
+      - Circulating Tumor Cells
+      - B-cell genotyping (BLGSP)
```
Comment (wwysoc2) [inline gdcdictionary/schemas/molecular_test.yaml]: Is the "BLGSP" just signifying which project asked for the change? @tlicht3

File: gdcdictionary/schemas/molecular_test.yaml (hunk 1 above)
Comment (tlicht3) [reply gdcdictionary/schemas/molecular_test.yaml]: yes - that was an obvious fix - I will make sure this is updated.

File: gdcdictionary/schemas/molecular_test.yaml (hunk 2 above)
Comment (tlicht3) [reply gdcdictionary/schemas/molecular_test.yaml]: yeah - this also needs updated.

**Pull request merged**